### PR TITLE
Intro chapter

### LINKED ISSUE #12: Add an introductory chapter
Why even have distributed work?

### PR DESCRIPTION
**Pull request by funnelfiasco**
First draft of the introduction. (Issue #12) It includes some discussion of the pre-release nature of the work that we can pretty easily edit for future pre-releases and then drop for the "final" release.

### CODE REVIEW

**Comment by funnelfiasco** [comment]
Forgot to squash my WIP commits. But they are now squashed.

File: 00_cotton-introduction.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+From there, we move into the more functional aspects of distributed work: communicating and collaborating.
+Before you can work together to refine ideas, you must be able to share them.
+This involves not only the *technology* you use for communicating and collaborating but also *the social norms* for *how* you use that technology.
+
+In the fourth section, we build the community for which the first three sections lay a foundation.
+Two aspects of community are important to distributed organizations.
+**// I'm not sure what line 66 means. Can you help readers understand more specifically what you're suggesting? —BB//**
+The first is growing your organization, both in number and in effectiveness.
+It also includes growing the bonds within your organization—helping people connect when they don't get to meet in physical space.
+
```
Comment (semioticrobotic) [inline 00_cotton-introduction.md]: I might suggest something like:

In the fourth section, we build the community for which the first three sections lay a foundation. Two aspects of community are important to distributed organizations. The first is the organization's ability to establish community among distributed teammates, no matter where they happen to be—creating shared bonds and collective goals that keep a far-flung band thinking and acting like a close-knit unit. The second is the organization's ability to *scale* its distributed teams by *growing* that sense of community—welcoming new members and weaving them into the shared values and experiences others have architected over time, without the benefit of co-located retreats, watercooler lore, and the like.

... something like that? Are those the two dimensions you're talking about?

**Comment by semioticrobotic** [comment]
Checking on your thoughts regarding my suggestions, @funnelfiasco. Don't know if you've had time to noodle (and if not, no worries).

**Comment by funnelfiasco** [comment]
@semioticrobotic I [said](https://github.com/open-organization/open-org-distributed-work-guide/issues/12#issuecomment-645556291) they looked good. Are you looking for something else?

I don't see the comments [you said you'd leave](https://github.com/open-organization/open-org-distributed-work-guide/issues/12#issuecomment-645621271)

**Comment by semioticrobotic** [comment]
Huh. Interesting. On my screen, I see (in this thread) reference to some inline comments I left in your file. It looks like this to me:

Screenshot from 2020-06-23 14-38-51

**Comment by funnelfiasco** [comment]
I don't. The "pending" tag suggests that it's waiting for you to push a button or something?

**Review by semioticrobotic — COMMENTED** [review]
Offering additional feedback on line 66.

**Comment by semioticrobotic** [comment]
> I don't. The "pending" tag suggests that it's waiting for you to push a button or something?

Well, well, well, if it isn't my own incompetence. Sorry, @funnelfiasco. I think I pushed the right button now.

**Comment by funnelfiasco** [comment]
The correct button has indeed been pushed. I like the general aim of your suggestion. I'll update with an edit by the end of the day.

**Comment by funnelfiasco** [comment]
@semioticrobotic edits pushed

**Review by semioticrobotic — APPROVED** [review]
Love it. Thank you!

**Pull request merged**